tdms/include/simulation_manager/fdtd_bootstrapper.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file fdtd_bootstrapper.h
+ * @brief Handles the boostrapping variables used in the FDTD solver.
+ *
+ * NOTE: Can be deleted from codebase without altering functionality. See https://github.com/UCL/TDMS/issues/214#issue-1532363539 - holding off doing so until meeting clarification.
```

Comment:
We forgot to ask about this?

Reply:
We did - although as it stands we can just "turn off" this part by deleting the member (and files associated to the structure).